src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -127,6 +134,26 @@ public class Utils {
     }
 
     /**
+     * Finds the parameterized type argument for a {@link MethodsToList} class and returns it.
+     *
+     * @param c A class.
+     * @return The parameterized type argument for the class, if it's a {@link MethodsToList} class. Null otherwise.
```

Comment:
Would be nice to have since Javadoc for such methods since they are explicitly allowed for use in Pipeline